libfaad/sbr_hfadj.c

```diff
@@ -301,26 +301,18 @@ static real_t find_log2_E(sbr_info *sbr, uint8_t k, uint8_t l, uint8_t ch)
         E >>= amp1;
         if (e < 0 || e >= 64 || E < 0 || E > 24)
             return LOG2_MIN_INF;
+        E -= 12;
 
-        if (ch == 0)
+        if (ch != 0)  // L/R anti-symmetry
```

Comment:
That's a lot better understandable now, thanks!

Reply:
YW